### PR TITLE
Added Azure to Az command table

### PR DESCRIPTION
**Pull request by JasonRShaver**

### CODE REVIEW

File: doc/azure2az_commands.rst [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+azure vm deallocate                                          az vm dellocate
+azure vm delete                                              az vm delete
+azure vm disk attach                                         az vm disk attach
+azure vm disk attach-new                                     az vm disk attach-new
+azure vm disk detach                                         az vm disk detach
+azure vm extension get                                       az vm extension get
+azure vm extension set                                       az vm extension set
+azure vm extension-image list                                az vm extension image list
+azure vm extension-image list-publishers                     az vm extension image list-names
+azure vm extension-image list-types                          az vm extension image list
```
Comment (tjprescott) [inline doc/azure2az_commands.rst]: So `vm extension image list` does the same work as two Xplat commands?

File: doc/azure2az_commands.rst [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+azure network vpn-gateway delete                             az network vpn-gateway delete
+azure network vpn-gateway list                               az network vpn-gateway list
+azure network vpn-gateway revoked-cert create                az network vpn-gateway revoked-cert create
+azure network vpn-gateway revoked-cert delete                az network vpn-gateway revoked-cert delete
+azure network vpn-gateway root-cert create                   az network vpn-gateway root-cert create
+azure network vpn-gateway root-cert delete                   az network vpn-gateway root-cert delete
+azure network vpn-gateway set                                az network vpn-gateway update
+azure network vpn-gateway show                               az network vpn-gateway show
+azure storage account check                                  az storage account check-name
+azure storage account connectionstring show                  az storage account connection-string
```
Comment (tjprescott) [inline doc/azure2az_commands.rst]: This is actually `az storage account show-connection-string` now.

File: doc/azure2az_commands.rst [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+azure storage table list                                     az storage table list
+azure storage table policy create                            az storage table policy create
+azure storage table policy delete                            az storage table policy delete
+azure storage table policy list                              az storage table policy list
+azure storage table policy set                               az storage table policy update
+azure storage table policy show                              az storage table policy show
+azure storage table sas create                               az storage table generate-sas
+azure vm capture                                             az vm capture
+azure vm create                                              az vm create
+azure vm deallocate                                          az vm dellocate
```
Comment (tjprescott) [inline doc/azure2az_commands.rst]: Typo: `az vm deallocate`

File: doc/azure2az_commands.rst [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+azure storage container lease break                          az storage container lease break
+azure storage container lease change                         az storage container lease change
+azure storage container lease release                        az storage container lease release
+azure storage container lease renew                          az storage container lease renew
+azure storage container list                                 az storage container list
+azure storage container policy create                        az storage container policy create
+azure storage container policy delete                        az storage container policy delete
+azure storage container policy list                          az storage container policy list
+azure storage container policy set                           az storage container policy update
+azure storage container policy show                          az storage container policy show
```
Comment (tjprescott) [inline doc/azure2az_commands.rst]: Similar to how you did for the DNS records, you could shorten this with:

`azure storage [container|table|queue] policy    az storage [container|table|queue] policy`

File: doc/azure2az_commands.rst [hunk 5]
```diff
@@ -0,0 +1,305 @@
...
+azure storage container sas create                           az storage container generate-sas
+azure storage container set                                  az storage container set-permission
+azure storage container show                                 az storage container show
+azure storage cors delete                                    az storage cors clear
+azure storage cors set                                       az storage cors add
```
Comment (tjprescott) [inline doc/azure2az_commands.rst]: `azure storage cors set` is NOT equivalent to `az storage cors add`. Xplat has you set the entire object graph in one ugly JSON encoded string. Az has you add entries to the CORS list one at a time using arguments.

File: doc/azure2az_commands.rst [hunk 6]
```diff
@@ -0,0 +1,305 @@
...
+azure storage cors show                                      az storage cors list
+azure storage directory create                               az storage directory create
+azure storage directory delete                               az storage directory delete
+azure storage file copy start                                az storage file copy start
+azure storage file copy stop                                 az storage file copy cancel
```
Comment (tjprescott) [inline doc/azure2az_commands.rst]: You should include: `azure storage file copy show    az storage file show`
People may wonder where this "missing command" is. The data is included in the get call, but Xplat suppresses it. We don't, so you only need to show that file/blob to see the copy status.

File: doc/azure2az_commands.rst [hunk 7]
```diff
@@ -0,0 +1,305 @@
...
+azure storage container lease release                        az storage container lease release
+azure storage container lease renew                          az storage container lease renew
+azure storage container list                                 az storage container list
+azure storage container policy create                        az storage container policy create
+azure storage container policy delete                        az storage container policy delete
+azure storage container policy list                          az storage container policy list
+azure storage container policy set                           az storage container policy update
+azure storage container policy show                          az storage container policy show
+azure storage container sas create                           az storage container generate-sas
+azure storage container set                                  az storage container set-permission
```
Comment (tjprescott) [inline doc/azure2az_commands.rst]: I think you might want to include our `az storage container show-permission` call. In Xplat, you use `azure storage container show` to get the container data PLUS the permission (it's a wonky server side thing). So customers used to Xplat may expect the permission to appear in show, but they need to use this other command instead. Not sure how you want to word that, but it's an important point for existing customers.

File: doc/azure2az_commands.rst [hunk 8]
```diff
@@ -0,0 +1,305 @@
...
+azure network vpn-gateway show                               az network vpn-gateway show
+azure storage account check                                  az storage account check-name
+azure storage account connectionstring show                  az storage account connection-string
+azure storage account create                                 az storage account create
+azure storage account delete                                 az storage account delete
+azure storage account keys list                              az storage account keys list
+azure storage account keys renew                             az storage account keys renew
+azure storage account list                                   az storage account list
+azure storage account sas create                             az storage account generate-sas
+azure storage account sas create                             az storage account generate-sas
```
Comment (tjprescott) [inline doc/azure2az_commands.rst]: Typo: you have this listed twice.

File: doc/azure2az_commands.rst (hunk 2 above)
Comment (JasonRShaver) [inline doc/azure2az_commands.rst]: fixed

File: doc/azure2az_commands.rst (hunk 8 above)
Comment (JasonRShaver) [inline doc/azure2az_commands.rst]: fixed

File: doc/azure2az_commands.rst (hunk 4 above)
Comment (JasonRShaver) [inline doc/azure2az_commands.rst]: I will fix this in a future revision

File: doc/azure2az_commands.rst (hunk 7 above)
Comment (JasonRShaver) [inline doc/azure2az_commands.rst]: fixed =)

File: doc/azure2az_commands.rst (hunk 5 above)
Comment (JasonRShaver) [inline doc/azure2az_commands.rst]: As long as this leads them to the right function area, I think we are fine.  I didn't want to match exact functionality.   I'll call this example (and concept) in our migration document I'm writing

File: doc/azure2az_commands.rst (hunk 6 above)
Comment (JasonRShaver) [inline doc/azure2az_commands.rst]: fixed

File: doc/azure2az_commands.rst (hunk 3 above)
Comment (JasonRShaver) [inline doc/azure2az_commands.rst]: fixed

File: doc/azure2az_commands.rst (hunk 1 above)
Comment (JasonRShaver) [inline doc/azure2az_commands.rst]: I was off a bit on this one; fixed.

File: doc/azure2az_commands.rst [hunk 9]
```diff
@@ -224,12 +224,14 @@ azure storage container policy show                          az storage containe
...
 azure storage container set                                  az storage container set-permission
 azure storage container show                                 az storage container show
+azure storage container show                                 az storage container show-permission
 azure storage cors delete                                    az storage cors clear
 azure storage cors set                                       az storage cors add
 azure storage cors show                                      az storage cors list
 azure storage directory create                               az storage directory create
 azure storage directory delete                               az storage directory delete
 azure storage file copy start                                az storage file copy start
+azure storage file copy show                                 az storage file start
```
Comment (tjprescott) [inline doc/azure2az_commands.rst]: az storage file show 😄

Comment (JasonRShaver) [inline doc/azure2az_commands.rst]: bah, fixed, thanks!

**Comment by tjprescott** [comment]
LGTM :shipit:

**Pull request merged**